src/uuid/builder/uuidutils.rs

```diff
@@ -44,8 +46,8 @@ impl UuidUtils {
     /// # Returns
     ///
     /// * `Result<Uuid, Error>` - UUID object built from the byte array
-    pub fn from_bytes(bytes: &[u8; 16]) -> Result<Uuid, Error> {
-        Ok(Uuid::from_bytes(*bytes))
+    pub fn from_bytes(bytes: &[u8; 16]) -> Result<uproto_Uuid, Error> {
```

Comment:
In Rust, usually this kind of operations are implemented via the `TryFrom` trait.

Reply:
done